MHVLib/Models/MHVModelBase.m

```diff
@@ -212,6 +212,11 @@ + (void)setProperties:(NSObject<MHVDataModelProtocol> *)object fromDictionary:(N
                             
                             @synchronized (propertyParametersCache)
                             {
+                                if (!objectParameters)
+                                {
+                                    objectParameters = [NSString class];
```

Comment:
This may have deeper implications. The problem now is that when compiled, objective C treats NSArray<NSString*> as just NSArray. It drops the type, so we don't have the object parameter type and string arrays end up as nil after deserialization. I'm not sure the deeper implications here - what happens of arrays of arrays of strings? Is this still ok for NSArray<NSNumber> ? I don't have good examples of those so I figured we'd cross that bridge when we come to it.